mainWindow.js

```diff
@@ -54,7 +54,7 @@ ipcMain.on('is-new-user', async (e, msg) => {
       e.sender.send('is-new-user', {error:"User already exists"})
       return
     }
-    await datastore.newUser(msg.email, msg.password)
+    await datastore.newUser(msg.email, msg.password, msg.username, msg.organization)
```

Comment:
@aviquez96 Maybe it would be a good idea to refactor the function signature!

What do you think about passing the object whole `msg` object to the `newUser` function?
That way we do not need too keep adding parameters, we will only modify:

```
var uInfoDoc = { 
                email: msg.email,
                password: msg.password,
                username: msg.username,
                organization: msg.organization,
                created_at: Math.round((new Date()).getTime() / 1000),
            };
```
for future changes

Reply:
Sounds good to me! On it